cluster-scope/base/core/namespaces/kvm-device-plugin/kustomization.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+components:
+    - ../../../../components/limitranges/default
+    - ../../../../components/resourcequotas/medium
```

Comment:
@LorbusChris would it make sense to give the `okd-wg` or `okd-team` ocp group access to this namespace, to verify the state of this daemonset, and ensure it's working as intended?

Reply:
yes, okd-team would make sense!

Reply:
updated